src/Components/Components/src/Rendering/ComponentState.cs

```diff
@@ -16,22 +15,13 @@ namespace Microsoft.AspNetCore.Components.Rendering
     /// </summary>
     internal class ComponentState
     {
-        private readonly int _componentId; // TODO: Change the type to 'long' when the Mono runtime has more complete support for passing longs in .NET->JS calls
-        private readonly ComponentState _parentComponentState;
-        private readonly IComponent _component;
         private readonly Renderer _renderer;
         private readonly IReadOnlyList<CascadingParameterState> _cascadingParameters;
         private readonly bool _hasAnyCascadingParameterSubscriptions;
-        private RenderTreeBuilder _renderTreeBuilderCurrent;
         private RenderTreeBuilder _renderTreeBuilderPrevious;
         private ArrayBuilder<RenderTreeFrame> _latestDirectParametersSnapshot; // Lazily instantiated
         private bool _componentWasDisposed;
 
-        public int ComponentId => _componentId;
```

Comment:
VS tells me to use auto properties instead

Reply:
@pranavkm Can you not do random cleanups :cry: It distracts from the main point